pkg/kubelet/cm/device_plugin_handler.go

```diff
@@ -166,7 +167,7 @@ func (h *DevicePluginHandlerImpl) Allocate(pod *v1.Pod, container *v1.Container,
 		resource := string(k)
 		needed := int(v.Value())
 		glog.V(3).Infof("needs %d %s", needed, resource)
-		if !deviceplugin.IsDeviceName(k) || needed == 0 {
+		if !v1helper.IsExtendedResourceName(k) || needed == 0 {
```

Comment:
Could you take a look at the current code that has changed to avoid using IsExtendedResourceName?